docs/api.md

```diff
@@ -553,12 +567,13 @@ Continuing the CT example, for a CT mirror personality (which must allow dupes s
 <a name="trillian.Proof"></a>
 
 ### Proof
-A consistency or inclusion proof for a Merkle tree. Output only.
+Proof holds a consistency or inclusion proof for a Merkle tree, as returned
+by the API.
 
 
 | Field | Type | Label | Description |
 | ----- | ---- | ----- | ----------- |
-| leaf_index | [int64](#int64) |  |  |
+| leaf_index | [int64](#int64) |  | leaf_index indicates the requested leaf index for leaf inclusion proof. This field is set to zero on a consistency proof. |
```

Comment:
preexisting: leaf_index == 0 is an entirely valid inclusion proof request and response, isn't it?  Context should probably be enough, but the wording here sounds a bit like a discriminator.

Reply:
Reworded.